docs/backtesting.md

```diff
@@ -79,12 +79,14 @@ Please also read about the [strategy startup period](strategy-customization.md#s
 
 Sometimes your account has certain fee rebates (fee reductions starting with a certain account size or monthly volume), which are not visible to ccxt.
 To account for this in backtesting, you can use `--fee 0.001` to supply this value to backtesting.
-This fee must be a percentage, and will be applied twice (once for trade entry, and once for trade exit).
+This fee must be a ratio, and will be applied twice (once for trade entry, and once for trade exit).
 
 ```bash
 freqtrade backtesting --fee 0.001
 ```
 
+!!! Note
+    Only supply this parameter if you want to experiment with different fee values. By default, Backtesting fetches the exchange's default fee from the exchange.
```

Comment:
"this parameter" --> "this option" or probably "this option (or corresponding configuration parameter)"/"this option (or corresponding configuration setting)"

"Backtesting fetches the default fee from the exchange." or "Backtesting fetches the default fee from the exchange pair/market info."